pack/trors.json

```diff
@@ -651,339 +651,231 @@
 		"resource_wild": 1,
 		"set_code": "hydra_camp",
-		"text": "Setup.<br>\n<b>Hero Action:</b> Discard this card and remove it from the campaign log → draw 5 cards.",
-		"threat_fixed": false,
+		"text": "Setup.\n<b>Hero Action:</b> Discard this card and remove it from the campaign log → draw 5 cards.",
 		"traits": "Tech.",
 		"type_code": "upgrade"
 	},
 	{
-		"base_threat_fixed": false,
 		"code": "04157",
 		"cost": 0,
 		"deck_limit": 1,
-		"double_sided": false,
-		"escalation_threat_fixed": false,
 		"faction_code": "campaign",
-		"health_per_hero": false,
-		"hidden": false,
-		"is_unique": false,
 		"name": "Emergency Teleporter",
 		"pack_code": "trors",
 		"position": 157,
 		"quantity": 1,
 		"resource_wild": 1,
 		"set_code": "hydra_camp",
-		"text": "Setup.<br>\n<b>Hero Action:</b> Discard this card and remove it from the campaign log → search your deck and discard pile for an ally, put it into play, and give it a tough status card.",
-		"threat_fixed": false,
+		"text": "Setup.\n<b>Hero Action:</b> Discard this card and remove it from the campaign log → search your deck and discard pile for an ally, put it into play, and give it a tough status card.",
 		"traits": "Tech.",
 		"type_code": "upgrade"
 	},
 	{
-		"base_threat_fixed": false,
 		"code": "04158",
 		"cost": 0,
 		"deck_limit": 1,
-		"double_sided": false,
-		"escalation_threat_fixed": false,
 		"faction_code": "campaign",
-		"health_per_hero": false,
-		"hidden": false,
-		"is_unique": false,
 		"name": "Laser Cannon",
 		"pack_code": "trors",
 		"position": 158,
 		"quantity": 1,
 		"resource_wild": 1,
 		"set_code": "hydra_camp",
-		"text": "Setup.<br>\n<b>Hero Action:</b> Discard this card and remove it from the campaign log → deal 5 damage to the villain and each enemy engaged with you.",
-		"threat_fixed": false,
+		"text": "Setup.\n<b>Hero Action:</b> Discard this card and remove it from the campaign log → deal 5 damage to the villain and each enemy engaged with you.",
 		"traits": "Tech.",
 		"type_code": "upgrade"
 	},
 	{
 		"back_link": "04159b",
-		"base_threat_fixed": false,
 		"code": "04159a",
 		"deck_limit": 1,
-		"double_sided": false,
-		"escalation_threat_fixed": false,
 		"faction_code": "campaign",
-		"health_per_hero": false,
-		"hidden": false,
-		"is_unique": false,
 		"name": "Basic Thwart Upgrade",
 		"pack_code": "trors",
 		"position": 159,
 		"quantity": 1,
 		"set_code": "hydra_camp",
-		"text": "Permanent. Setup.<br>\nYou get +2 hit points.<br>\nYour hero gets +1 THW.",
-		"threat_fixed": false,
+		"text": "Permanent. Setup.\nYou get +2 hit points.\nYour hero gets +1 THW.",
 		"traits": "Condition.",
 		"type_code": "upgrade"
 	},
 	{
-		"base_threat_fixed": false,
 		"code": "04159b",
 		"deck_limit": 1,
-		"double_sided": false,
-		"escalation_threat_fixed": false,
 		"faction_code": "campaign",
-		"health_per_hero": false,
-		"hidden": true,
-		"is_unique": false,
 		"name": "Improved Thwart Upgrade",
+    "hidden": true,
```

Comment:
**nit pick** It seems the indentation on the new `hidden` properties is a bit off compared to the existing ones.